### DISCUSSION TITLE
Do you still consider smart_open as in beta stage?

### DISCUSSION THREAD

**Discussion by seebi** [RaRe-Technologies/smart_open#797] [General]
Hi all,

according to https://github.com/piskvorky/smart_open/blob/develop/setup.py#L89 the project is classified as in beta stage.

Based on other metrics (e.g. amount of releases, etc) I think its more than that.

As maintainers, do you have an opinion on this?

best @seebi

cc: @msaipraneeth

**Comment by piskvorky** [comment]
No, `smart_open` has been stable for a long time. Nobody bothered / noticed to update `setup.py` – that particular line is 10 years old :)

Can you open a PR @seebi?

**Reply by seebi** [reply]
https://github.com/piskvorky/smart_open/pull/798
